src/resources/PanelLayout.py

```diff
@@ -19,6 +19,10 @@ def make_layout(self, start: bool) -> Layout:
                 Layout(name='main_game', ratio=7),
                 Layout(name='footer', ratio=2)
             )
+            layout['footer'].split_row(
```

Comment:
I just pushed a recent commit to fix a merge conflict, such as this, great work though!